src/apps/relay/dbdrivers/dbd_sqlite.c

```diff
@@ -194,15 +191,12 @@ static void init_sqlite_database(sqlite3 *sqliteconnection) {
       "CREATE TABLE admin_user (name varchar(32), realm varchar(127), password varchar(127), primary key (name))",
       NULL};
 
-  int i = 0;
-  while (statements[i]) {
+  for (int i = 0; statements[i] != NULL; ++i) {
     sqlite3_stmt *statement = NULL;
-    int rc = 0;
-    if ((rc = sqlite3_prepare(sqliteconnection, statements[i], -1, &statement, 0)) == SQLITE_OK) {
+    if (sqlite3_prepare(sqliteconnection, statements[i], -1, &statement, 0) == SQLITE_OK) {
```

Comment:
int rc is written to but never read from.

This pattern repeats throughout the whole file.